source/manual/fall-back-to-mirror.html.md

```diff
@@ -20,7 +20,7 @@ If Fastly goes down, we would manually [switch to AWS CloudFront](https://docs.p
 
 ## GOV.UK mirror locations and access
 
-The GOV.UK mirror is hosted in an [Amazon Web Services (AWS) S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). The bucket contains copies of GOV.UK HTML files. The mirror is static, meaning dynamic pages such as search pages will not work.
+The GOV.UK mirror is hosted in an [Amazon Web Services (AWS) S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). The bucket contains copies of GOV.UK HTML files. The mirror is static, meaning dynamic pages such as search pages will not work. Search and non-existent pages return 403 Forbidden response.
```

Comment:
This is a bit ambiguous and potentially misleading - it's unclear to the reader whether this is saying that the 403 is intended behaviour from the end-user perspective when using the mirrors (it's not) or whether it's talking about just the S3->Fastly bit (i.e. an implementation detail).

If we're not already changing these 403s to 404s in Fastly when serving from mirrors, then that sounds like something we should fix rather than document.

Reply:
I'd recommend clarifying that the 403 is an S3 thing, for example something like: `S3 will return `403 Forbidden` for non-existent pages instead of 404, because we don't allow ListBucket permission. This also affects requests for features such as Search, which are not covered by the mirrors.`

Reply:
Thanks, that's a very good point! I'll change it to your description 🌟